backend/src/v4/models/risk.js

```diff
@@ -129,6 +131,16 @@ class Risk extends Ticket {
 		}
 
 		newRisk = await super.create(account, model, newRisk);
+
+		try{
+			const teamspaceSettings = await TeamspaceSetting.getTeamspaceSettings(account,	{  _id: 0, createTreatmentSuggestions: 1});
```

Comment:
Can we rename this to `createMitigationSuggestions`

We only use the term `Treatment` on the UI

Reply:
I think this part should go into mitigations.

It's up to mitigations to determine if something should be added or not, not risk

Let mitigations pull the teamspace settings and check